test/bin/scenario.sh

```diff
@@ -196,7 +199,20 @@ prepare_kickstart() {
               -e "s|REPLACE_REDHAT_AUTHORIZED_KEYS|${REDHAT_AUTHORIZED_KEYS}|g" \
               -e "s|REPLACE_PUBLIC_IP|${PUBLIC_IP}|g" \
               -e "s|REPLACE_FIPS_COMMAND|${fips_command}|g" \
+              -e "s|REPLACE_ENABLE_MIRROR|${ENABLE_REGISTRY_MIRROR}|g" \
               > "${output_file}"
+    if ${ENABLE_REGISTRY_MIRROR}; then
+        if [ ! -f "${cert}" ]; then
+            error "No ${cert} was found. Did you run mirror_registry.sh?"
+            record_junit "${vmname}" "prepare_kickstart" "no-mirror-cert"
```

Comment:
```suggestion
            record_junit "${vmname}" "prepare_kickstart" "FAILED"
```

I do not think junit accepts anything but OK / FAIL / SKIP

Reply:
It defaults to FAILED if its not OK/SKIP*: https://github.com/openshift/microshift/blob/main/test/bin/scenario.sh#L339-L342